docs/src/prototypes/employeeCard/Text.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react'
+import { Text as StardustUIText } from '@stardust-ui/react'
+
+const Text = props => {
+  const { muted, ...rest } = props
+  return <StardustUIText {...rest} styles={{ ...(muted && { color: '#888' }) }} />
```

Comment:
if `StardustUIText` is an only option here, then this is an important point to reflect as a concern to address. Lets mention it in the PR's description